beetsplug/replaygain.py

```diff
@@ -66,8 +69,10 @@ def call(args):
 
 
 class Backend(object):
+
     """An abstract class representing engine for calculating RG values.
     """
+
```

Comment:
This empty line is fine, though—it's actually in line with PEP 8 to have a blank line *after* docstrings in class declarations.